problem_builder/tests/integration/xml_templates/feedback_persistence_mcq_general_feedback.xml

```diff
@@ -0,0 +1,11 @@
+<vertical_demo>
+    <problem-builder url_name="mcq_general_feedback" enforce_dependency="false">
```

Comment:
@mtyaka Nit: Can we do without `enforce_dependency="false"` in the templates that are used by `test_feedback_persistence_tips`? It seems to be redundant since `enforce_dependency` has a default value of `False`.